packages/node_modules/pouchdb-abstract-mapreduce/src/index.js

```diff
@@ -285,7 +285,17 @@ function createAbstractMapReduce(localDocName, mapper, reducer, ddocValidator) {
         method: method,
         url: '_design/' + parts[0] + '/_view/' + parts[1] + params,
         body: body
-      }).then(postprocessAttachments(opts));
+      }).then(function (result) {
+        // fail the entire request if the result contains an error
+        var reduceError = result.rows.find(function (row) {
+          return row.value && row.value.error && row.value.error === "builtin_reduce_error";
+        });
+        if (reduceError) {
+          throw new Error(reduceError);
```

Comment:
The string is an implementation detail of CouchDB (master). I think it would be appropriate to move into `pouchdb-errors` if we expected pouchdb to surface it directly or used it for a string comparison in a test but in this case we're suppressing it.